### PR TITLE
Added Grid Component 

### PR DESCRIPTION
**Pull request by Manpreet-hub**
Added Grid Component 
4hsvjCw8Jm

### CODE REVIEW

**Review by ishapatell1 — APPROVED** [review]
Grid Code is good to merge.

File: Components/grid/grid.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+.three-item-grid{
+    grid-template-columns: repeat(3,1fr);
+}
+
+.grid-item{
+    background-color: var(--voilet-color);
+}
+
+@media screen and (max-width:580px){
```
Comment (shivanigangadharan) [inline Components/grid/grid.css:20]: Just a suggestion, can also add the following :- 
`@media screen and (max-width:992px){
    .three-item-grid{
        grid-template-columns: repeat(2, 1fr);
    }
}
`

**Review by shivanigangadharan — APPROVED** [review]
Looks good to merge. 👍

File: Components/grid/grid.css (hunk 1 above)
Comment (Manpreet-hub) [reply Components/grid/grid.css:20]: Done [ee3ef13](https://github.com/Manpreet-hub/ComponentLibrary/pull/16/commits/ee3ef13e7d2f0691fe4d4d8699aa802658ad521d)

**Pull request merged**